RadixWallet/Features/CreatePersona/Children/Introduction/IntroductionToPersonas.swift

```diff
@@ -4,22 +4,13 @@ import SwiftUI
 @Reducer
 public struct IntroductionToPersonas: Sendable, FeatureReducer {
 	@ObservableState
-	public struct State: Sendable, Hashable {
-		@Presents
-		public var infoPanel: SlideUpPanel.State?
-		public init() {}
-	}
+	public struct State: Sendable, Hashable {}
```

Comment:
minor: I would move this into a vanilla SwiftUI view and remove the reducer just to inform delegate.